scripts/test/shared.py

```diff
@@ -151,7 +151,11 @@ def is_exe(fpath):
         if is_exe(program):
             return program
     else:
-        for path in os.environ["PATH"].split(os.pathsep):
+        paths = os.environ["PATH"].split(os.pathsep)
```

Comment:
Can you add the PATH paths after the other ones to avoid having to insert paths at the beginning of the list?